uiowa/uiowa
# Card styles for all the things!

# To Test

This extends card style options to events, articles, people blocks within v3. Styles should control child items. Generally, basic use of the website as well as the update hooks should not present errors.

uids_base style clean up will be saved for another day

## HR

- This PR should not affect HR's website. Article, Events, People listings should look the same as it is sub-themed off uiowa_bootstrap.

## v2 Websites (e.g. hawkeyemarchingband, diversity, law, brand)

- Check SiteNow Articles/People view pages including the People manual sort option. Should look the same.
- Check list/grid/masonry style options for events, articles, people, featured content paragraphs. Should look the same.

## v3 Websites (e.g. sandbox, uiowa, admissions, grad, international)

- Existing layout blocks should be set with PROD defaults and look similar:
  - Articles: left-aligned content, right-aligned small widescreen image
  - Events: left-aligned content, left-aligned small circle image with border
  - People: left-aligned content, right-aligned small square image
- New layout blocks should have the following defaults with the option labels showing (default) appended.
  - Articles: White background, left-aligned content, right-aligned small widescreen image
  - Events/People: White background, left-aligned content, left-aligned small circle image with border
- Changes between widescreen image and square/circle should change the image view mode and clear necessary caches.
- SiteNow People/Articles view pages should be unchanged for now and should look similar to a v2 website,  including the People manual sort option.
